SendGrid/Example/Example.cs

```diff
@@ -35,13 +35,13 @@ private static void Main()
             String subject = "I'm replacing the subject tag";
             Email to = new Email("elmer@sendgrid.com");
             Content content = new Content("text/html", "I'm replacing the <strong>body tag</strong>");
-            Mail mail = new Mail(from, subject, to, content);
+            SendGrid.Models.Mail mail = new SendGrid.Models.Mail(from, subject, to, content);

```

Comment:
I agree to inject the Api client as a dependency, but I think it should be injected in the EmailService not in the Email object. In my opinion, Email object should be just a DTO.